clang/lib/CodeGen/CGLoopInfo.cpp

```diff
@@ -440,6 +440,14 @@ MDNode *LoopInfo::createMetadata(
         Ctx, {MDString::get(Ctx, "llvm.loop.parallel_accesses"), AccGroup}));
   }
 
+  // Setting clang::code_align attribute.
+  if (Attrs.CodeAlign > 0) {
```

Comment:
Where is this consumed?

Reply:
Added consumption on this IR in MBBPlacement: https://github.com/llvm/llvm-project/pull/71026 by @FreddyLeaf